catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/EntityDAO.java

```diff
@@ -52,6 +52,9 @@
   @SqlQuery("SELECT json FROM <table> WHERE <nameColumn> = :name AND deleted IS NOT TRUE")
   String findByName(@Define("table") String table, @Define("nameColumn") String nameColumn, @Bind("name") String name);
 
+  @SqlQuery("SELECT json FROM <table> WHERE <nameColumn> = :name")
+  String findByNameDeletedOrAlive(@Define("table") String table, @Define("nameColumn") String nameColumn, @Bind("name") String name);
```

Comment:
Minor nit, may be use "DeletedOrExists" instead of Alive. But I am ok either way